docs/core_docs/src/theme/FeatureTables.js

```diff
@@ -261,6 +261,13 @@ const FEATURE_TABLES = {
         apiLink:
           "https://api.python.langchain.com/en/latest/llms/langchain_ai21.llms.AI21LLM.html#langchain_ai21.llms.AI21LLM",
       },
+      {
+        name: "Arcjet Redact",
+        link: "arcjet",
+        package: "langchain-arcjet",
+        // apiLink:
```

Comment:
Was unsure what to do with this apiLink field as we don't have a Python version of this package.